docs/pages/release_notes.md

```diff
@@ -22,3 +22,5 @@ This is a {{ site.pmd.release_type }} release.
 
 {% endtocmaker %}
 
+{% include note.html content="The release notes of previous versions are available [here](pmd_release_notes_old.html)" %}
```

Comment:
Let's see, how this turns out in a non-jekyll environment like github's release - our release script uploads the release notes to github and as a readme markdown to sourceforge. Maybe we need to get rid of the nice box...